content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -106,14 +107,14 @@ spec:
 
 {{< note >}}
 
-In the case, the label `app=nginx` is automatically added.
+NetworkPolicy includes a `podSelector` which selects the grouping of Pods to which the policy applies. Our policy selects Pods with the label `app=nginx`. This label was automatically added to the Pod in our nginx Deployment. An empty `podSelector` selects all pods in the namespace.
```

Comment:
> Our policy selects&hellip;

Ideally: [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you).
For example, “This policy selects&hellip;”

Reply:
Good suggestion. I have reworded the paragraph and I think now it reads better. Thanks.